knowledge/style-guide.md

```diff
@@ -15,94 +15,136 @@ parent: Knowledge
+This document will present the officially supported Bedrock-Wiki style-guide for addon-creation. The goal of this guide is to promote best-practices
+while creating addons and create a consistent format for everyone to follow.
 
-The style-guide is a living, breathing document, which will evolve as addon-creation evolves. Please get in touch if you think something needs to be updated ot changed. 
+The style-guide is a living, breathing document, which will evolve as addon-creation evolves. Please get in touch if you think something needs to be
+updated or changed.
 
 ## Folder Structure
- - No spaces in your file paths. `use_underscores`.
- - No `CAPITALS` in your identifiers, file names, or folder names, except at the top level.
- - The total character-length of any path must not exceed 80 characters (console limitation).
- - Content folders should use consistent pluralization: Don't mix and match. 
+
+- No spaces in your file paths. `use_underscores`.
+- No `CAPITALS` in your identifiers, file names, or folder names.
+- The total character-length of any path must not exceed 80 characters (console limitation).
+- Content folders should use consistent pluralization: Don't mix and match.
 
 ## Identifiers
-Do not use identifiers that begin with a number, and especially don't use an identifier that is *only* a number. This applies to entities, component_groups, events, and anything else that takes a `namespace:name` pair.
+
+Identifiers are used almost everywhere, and this section applies to entities, component_groups, events, and anything else that takes a
+`namespace:name` pair. Identifiers are to be specified using the following rules:
+
+- Do not start the identifier with a number.
+- Identifiers cannot be numbers only.
+- All lowercase.
+- No spaces.
 
 ## File and Folder names
 
-| Concept              | Example Identifier         |
-|----------------------|----------------------------|
-| Behavior Pack        | dragons_BP                 |
-| Resource Pack        | dragons_RP                 |
-| Geometry             | dragon.geo.json            |
-| Animation            | dragon.animation.json      |
-| Animation Controller | dragon.ac.json             |
-| RP Entity            | dragon.entity.rp.json      |
-| BP Entity            | dragon.entity.bp.json      |
-| BP Item              | dragon_tooth.item.bp.json  |
-| RP Item              | dragon_tooth.item.rp.json  |
-| Render Controller    | dragon.render.json         |
-| Loot Table           | dragon.loot.json           |
-| Dragon Saddle        | dragon_saddle.recipe.json  |
-| Spawn Rules          | dragon.spawn.json          |
-| Trade Table          | dragon.trade.json          |
-| Particles            | dragon_magic.particle.json |
-| Texture              | dragon.png                 |
+If multiple options are aviable the prefence go to the first found item, however the second one is recognized as valid as well.
```

Comment:
Misspelled aviable

Reply:
Of all the things I put through Grammarly, I forgot that one :(